django_scheduled_tasks/models.py

```diff
@@ -15,14 +15,25 @@ class ScheduledTask(models.Model):
     """A record represents a single scheduled (recurring) task."""
 
     func = models.CharField(max_length=100, unique=True)  # importable function name
-    interval_minutes = models.IntegerField(validators=[MinValueValidator(1),
-                                                       MaxValueValidator(1440)])  # 1 min to 24 hours
+
     onstart = models.BooleanField(default=False)  # run at django startup?
     enabled = models.BooleanField(default=True)
     exclusive = models.BooleanField(default=True)  # run as the only task (I.e. thread locked)
+
     last_timestamp = models.DateTimeField(null=True, blank=True)
     last_success = models.BooleanField(null=True, blank=True)
     last_runtime = models.FloatField(null=True, blank=True)
+    day = models.CharField(max_length=3, null=True, blank=True)  # Day of the week ('mon', 'tue', etc.)
```

Comment:
Can you validate this with choices? https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices